src/main/java/com/thevoxelbox/voxelsniper/brush/type/RulerBrush.java

```diff
@@ -20,35 +24,55 @@
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        for (String parameter : parameters) {
-            if (parameter.isEmpty()) {
-                continue;
-            }
-            if (parameter.equalsIgnoreCase("info")) {
-                messenger.sendMessage(ChatColor.GOLD + "Ruler Brush instructions: Right click first point with the arrow. Right click with powder for distances from that block (can repeat without getting a new first block.) For placing blocks, use arrow and input the desired coordinates with parameters.");
-                messenger.sendMessage(ChatColor.LIGHT_PURPLE + "/b r x[x value] y[y value] z[z value] -- Will place blocks one at a time of the type you have set with /v at the location you click + this many units away.  If you don't include a value, it will be zero.  Don't include ANY values, and the brush will just measure distance.");
-                messenger.sendMessage(ChatColor.BLUE + "/b r ruler -- will reset the tool to just measure distances, not layout blocks.");
-                return;
-            } else if (parameter.charAt(0) == 'x') {
-                this.offsetX = Integer.parseInt(parameter.replace("x", ""));
+        String firstParameter = parameters[0];
+
+        if (firstParameter.equalsIgnoreCase("info")) {
+            messenger.sendMessage(ChatColor.GOLD + "Ruler Brush instructions: Right click first point with the arrow. " +
+                    "Right click with gunpowder for distances from that block (can repeat without getting a new first " +
+                    "block.) For placing blocks, use arrow and input the desired coordinates with parameters.");
+            messenger.sendMessage(ChatColor.LIGHT_PURPLE + "/b r [x] [y] [z] -- Places blocks one at a time of the type you " +
```

Comment:
Using 2 args will cause an AIOOBE. Using 3 args in the following format `/b r 50 50 50` complaints about the format.